log-collector-script/linux/eks-log-collector.sh

```diff
@@ -534,7 +534,7 @@ get_networking_info() {
     CA_CRT=$(grep certificate-authority: "${COLLECT_DIR}"/kubelet/kubeconfig.yaml | sed 's/.*certificate-authority: //')
     for i in $(seq 5); do
       echo -e "curling ${API_SERVER} ($i of 5) $(date --utc +%FT%T.%3N%Z)\n\n" >> ${COLLECT_DIR}"/networking/curl_api_server.txt"
-      timeout 75 curl -v --cacert "${CA_CRT}" "${API_SERVER}"/livez?verbose >> ${COLLECT_DIR}"/networking/curl_api_server.txt" 2>&1
+      timeout 75 curl -v --connect-timeout 3 --no-proxy '*' --cacert "${CA_CRT}" "${API_SERVER}"/livez?verbose >> ${COLLECT_DIR}"/networking/curl_api_server.txt" 2>&1
```

Comment:
it times out at 75 seconds in my environment, with no external connectivity.

Reply:
as described in description behaviour is.
> connect-timeout 3 to reduce total execution time. Otherwise stalls for duration of standard timeout of 75 seconds, which runs in a loop of 5 times. So total elapsed time of 375 seconds.

Reply:
```
time curl -Lv --noproxy '*'  https://10.255.255.1
*   Trying 10.255.255.1:443...
* connect to 10.255.255.1 port 443 failed: Connection refused
* Failed to connect to 10.255.255.1 port 443 after 75001 ms: Couldn't connect to server
* Closing connection 0
curl: (7) Failed to connect to 10.255.255.1 port 443 after 75001 ms: Couldn't connect to server

real	1m15.010s
user	0m0.008s
sys	0m0.009s
```

```
bash-4.2# time curl --connect-timeout 3 -Lv --noproxy '*'  https://10.255.255.1
*   Trying 10.255.255.1:443...
* ipv4 connect timeout after 3000ms, move on!
* Failed to connect to 10.255.255.1 port 443 after 3000 ms: Timeout was reached
* Closing connection 0
curl: (28) Failed to connect to 10.255.255.1 port 443 after 3000 ms: Timeout was reached

real	0m3.009s
user	0m0.004s
sys	0m0.005s
```

```
bash-4.2# time curl --connect-timeout 3 --max-time 10 -Lv --noproxy '*'  https://10.255.255.1
*   Trying 10.255.255.1:443...
* ipv4 connect timeout after 3000ms, move on!
* Failed to connect to 10.255.255.1 port 443 after 3001 ms: Timeout was reached
* Closing connection 0
curl: (28) Failed to connect to 10.255.255.1 port 443 after 3001 ms: Timeout was reached

real	0m3.011s
user	0m0.008s
sys	0m0.003s
```

```
time curl --connect-timeout 1 --max-time 0.01 -Lv --noproxy '*'  https://google.com
*   Trying 142.250.187.238:443...
* Connection timed out after 11 milliseconds
* Closing connection 0
curl: (28) Connection timed out after 11 milliseconds

real	0m0.020s
user	0m0.007s
sys	0m0.004s
```